troubleshooting/debugging.md

```diff
@@ -1498,5 +1504,114 @@ If you use Amazon ECS FireLens, then standard out and standard error will be col
+
+There are test cases at different throughputs, with some running at [10MB/s sustained throughput](https://github.com/aws/firelens-datajet/blob/6eaeb5f8e15739e1caf96e2c61236912502da2a3/apps/firelens-stability/collections/ecs-firelens-stability-tests/golden-path/cases/onepod-high-throughput.json), which is much higher than most real world use cases.
+
+Our CloudWatch Logs tests use a local network mock of the CloudWatch API; thus, these tests are not fully real world as there will be less network latency sending to the mock. 
+
+We also run a number of tests against the S3 output which do send to Amazon S3:
+- [fluent-bit.conf](https://github.com/aws/firelens-datajet/blob/6eaeb5f8e15739e1caf96e2c61236912502da2a3/apps/firelens-stability/templates/s3-fargate-v04-05-2023/fluent-bit.conf)
+
+One of the S3 test cases runs at [30MB/s sustained throughput](https://github.com/aws/firelens-datajet/blob/6eaeb5f8e15739e1caf96e2c61236912502da2a3/apps/firelens-stability/collections/ecs-firelens-stability-tests/s3-stability/cases/s3-throughput-30mbps.json). 
+
+All of tests run on Fargate and are provisioned with [1 vCPU and 4 GB of memory](https://github.com/aws/firelens-datajet/blob/6eaeb5f8e15739e1caf96e2c61236912502da2a3/apps/firelens-stability/templates/golden-path-mountebank-fargate-v01-11-2023/task-definition.json#L159). 
+
+Finally, our tests mostly configure inputs with the default `storage.type memory` and with a [Mem_Buf_Limit 64M](https://github.com/aws-samples/amazon-ecs-firelens-examples/tree/mainline/examples/fluent-bit/oomkill-prevention).
+
+![graph of memory usage](images/stability_test_memory.png)
+
+We see memory usage vary considerably across test cases based on throughput, complexity of the configuration, and other unknown factors. We also typically see that memory can increase slightly over the lifetime of the Fluent Bit container, however, this does not by itself indicate a [memory leak](#memory-leaks-or-high-memory-usage).
+
+Memory usage for the lower throughput tests tends to vary around ~40MB to ~100MB, with high throughput tests consuming up to 400MB of memory or more. 
+
+![graph of CPU usage](images/stability_test_cpu.png)
+
+CPU usage varies considerably, especially over time within a single test case. ~25+% of 1vCPU is typical even for low throughput test runs, and CPU usage can easily be sustained at ~50% vCPU, with spikes at or above 1vCPU. In one past experiment, we saw [consistent CPU spikes once a day](https://github.com/fluent/fluent-bit/issues/6503) which we were not able to root cause. 
+
+The lesson here is that CPU usage can vary considerably.
+
+#### Recommendations for high throughput
+
+First of all, we must define "high throughput". The AWS Distro for Fluent Bit team generally considers a single Fluent Bit instance sending at a sustained rate of over 1MB/s to be "high throughput". Most Fluent Bit instances send at sustained rates that are below this. 
```

Comment:
AWS for fluent bit release notes (https://github.com/aws/aws-for-fluent-bit/releases) shows sending logs 20 MB/s to Kinesis, S3. Why we say sending logs 1 MB/s is high throughput. Do I miss anything?